### PR TITLE
Review request for filter phase II labs (114) on Vlabs landing pages

### PR DESCRIPTION
**Pull request by sravanthimodepu**
@travula, @ksripathi, @madhavipuliraju 
Please review the code and let us know if any changes are required.

### CODE REVIEW

File: src/browser-compatibility-for-landing-pages.org [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#+TITLE: Plan for vlabs landing pages regarding support for firefox browser
```
Comment (travula) [inline src/browser-compatibility-for-landing-pages.org]: Remove this file from this  repository and put it in the repository that deals with creating a browser compatible for virtual labs

File: src/runtime/controller.org [hunk 2]
```diff
@@ -811,6 +814,19 @@ var displayAllLabs = function() {
 };
 
 #+END_SRC
+* Filter Phase-2 Labs
+  This function filters phase2 labs (114) from list of all labs (179).
+#+name: display-phase-2-labs
+#+BEGIN_SRC js
+
+var filterByPhase2Labs = function(labs) {
+  return labs.filter(function(labs) { return ((labs.phase.length == phaseLength) && (labs.phase[0].phase_id == phaseId)); })
```
Comment (travula) [inline src/runtime/controller.org]: Ensure all labs are returned when variable phaseLength is set to 0

Comment (travula) [reply src/runtime/controller.org]: Also, where is the configuration setting.

File: src/runtime/controller.org [hunk 3]
```diff
@@ -811,6 +814,19 @@ var displayAllLabs = function() {
 };
 
 #+END_SRC
+* Filter Phase-2 Labs
+  This function filters phase2 labs (114) from list of all labs (179).
```
Comment (travula) [inline src/runtime/controller.org]: Write the narrative to fit what is being achieved.

**Comment by mrudhvika940** [comment]
@travula 
Changes are made as per the review comments.
Please review and let me know if any more changes are required.

**Pull request merged**